src/main/java/Strategy/GameInitStrategy.java

```diff
@@ -2,8 +2,15 @@
 
 import Builder.GameBuilder;
 import Validators.ValidatorFacade;
+import model.Connector;
+
+import java.util.HashMap;
 
 public interface GameInitStrategy {
 
     public GameBuilder initializeGame(GameBuilder gameBuilder, ValidatorFacade validatorFacade);
+
```

Comment:
No need to add if these methods are private, we do not want anyone to access these methods outside the gameInitStrategy functions to avoid the game going into any unexpected state.

Reply:
I thought that if we want to add another element apart from snakes and ladders, then we will have to initialize them in every initstrategy so I put those methods in the interface itself.

Reply:
I can either make these methods private or I can keep them in the interface. Is there a way to do both?

Reply:
Make those methods Private. If we need to add another component we can add another interface and make our concrete classes extend that interface.